test/test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"testing"
 	"time"
 
-	_ "cloud.google.com/go"
+	// _ "cloud.google.com/go"
```

Comment:
@jkralik do you recall why this is needed? `golangsci-lint` complained about it, saying that I should either remove it or add a comment why.

```
test/test.go:28:2: blank-imports: a blank import should be only in a main or test package, or have a comment justifying it (revive)
        _ "cloud.google.com/go"
        ^
```